posthog/batch_exports/service.py

```diff
@@ -643,6 +643,11 @@ def sync_batch_export(batch_export: BatchExport, created: bool):
 
     destination_config_fields = {field.name for field in fields(workflow_inputs)}
     destination_config = {k: v for k, v in batch_export.destination.config.items() if k in destination_config_fields}
+    task_queue = (
+        BATCH_EXPORTS_TASK_QUEUE
+        if batch_export.destination.type in ("BigQuery", "Redshift")
+        else SYNC_BATCH_EXPORTS_TASK_QUEUE
+    )
```

Comment:
This means whenever we update a batch export for BigQuery or Redshift (or a new one is created) they will start being routed to the new queue.